llvm/test/CodeGen/AMDGPU/lower-module-lds-force-table.ll

```diff
@@ -0,0 +1,264 @@
+; RUN: opt -S -mtriple=amdgcn-- -mcpu=gfx942 -amdgpu-lower-module-lds < %s 2>&1 | FileCheck %s
+; RUN: opt -S -mtriple=amdgcn-- -mcpu=gfx942 -passes=amdgpu-lower-module-lds < %s 2>&1 | FileCheck %s
+
```

Comment:
This test isn't reduced enough. I previously posted a godbolt link which is smaller than this, and I'm sure it can be shrunk further

Reply:
I see the godbolt IR. In what sense is this believed to change allocated lds size?